cmd/addvalidators_internal_test.go

```diff
@@ -104,7 +104,6 @@ func TestValidateConfigAddValidators(t *testing.T) {
 	}
 }
 
-// TODO(xenowits): Add more extensive tests, this is just a very simple unit test.
```

Comment:
The test is already comprehensive enough to test the "add validators" feature

In addition to that, the add validators feature is not in production, it is under the `charon alpha` command